<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HanziFeatures</class>
 <widget class="QDialog" name="HanziFeatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>170</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Hanzi Features</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineGroupsLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="insideSpacesLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lineGroupsLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Line groups</string>
   </property>
  </widget>
  <widget class="QLabel" name="insideSpacesLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inside spaces</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
